controllers/api/booking-routes.js

```diff
@@ -0,0 +1,72 @@
+const router = require("express").Router();
+const { Booking, User, Service } = require("../../models");
+
+// GET all bookings 
+
+router.get ('/', withAuth, async (req,res) => {
+try {
+    const bookingData = await Booking.findAll({
+        include: [{ model: User, Service }],
```

Comment:
```suggestion
        include: [ User, Service ],
```
or 
```suggestion
        include: [{model: User}, {model: Service}]
```


I think we can do it like this: https://sequelize.org/api/v6/class/src/model.js~model#static-method-findAll